diff -Naur root.as-needed/bindings/pyroot/Module.mk root/bindings/pyroot/Module.mk
--- root.as-needed/bindings/pyroot/Module.mk	2017-03-14 20:21:27.000000000 +0700
+++ root/bindings/pyroot/Module.mk	2017-03-02 20:47:28.000000000 +0700
@@ -55,8 +55,6 @@
 	@[ -d $(dir $@) ] || mkdir -p $(dir $@)
 	cp -R $< $@
 endif
-ROOTPYC      := $(ROOTPY:.py=.pyc)
-ROOTPYO      := $(ROOTPY:.py=.pyo)
 ROOTAASC     := $(ROOTAAS:.py=.pyc)
 ROOTAASO     := $(ROOTAAS:.py=.pyo)
 
@@ -74,10 +72,7 @@
 include/%.h:    $(PYROOTDIRI)/%.h
 		cp $< $@
 
-%.pyc: %.py;    python -c 'import py_compile; py_compile.compile( "$<" )'
-%.pyo: %.py;    python -O -c 'import py_compile; py_compile.compile( "$<" )'
-
-$(PYROOTLIB):   $(PYROOTO) $(PYROOTDO) $(ROOTPY) $(ROOTPYC) $(ROOTPYO) \
+$(PYROOTLIB):   $(PYROOTO) $(PYROOTDO) $(ROOTPY) \
                 $(ROOTLIBSDEP) $(PYTHONLIBDEP) \
                 $(ROOTAAS) $(ROOTAASC) $(ROOTAASO)
 
@@ -121,7 +116,7 @@
 
 distclean-$(MODNAME): clean-$(MODNAME)
 		@rm -f $(PYROOTDEP) $(PYROOTDS) $(PYROOTDH) $(PYROOTLIB) \
-		   $(ROOTPY) $(ROOTPYC) $(ROOTPYO) $(PYROOTMAP) \
+		   $(ROOTPYO) $(PYROOTMAP) \
 		   $(PYROOTPYD)
 		@rm -rf $(LPATH)/ROOTaaS bin/ROOTaaS
 
